### PR TITLE
fix for poetry funbox word limit

### LINKED ISSUE #1721: Progress & Poetry Funbox
Yes I did clear my cache before this 

**Describe the bug**  You should be able to find it in the "reproduce" and expected result

 I did not try it on incognito mode as the conditions I'm in don't really allow me to try that.

**To Reproduce** 

1. Set your timer/progress to "mini" or "text"
2. Set your funbox to "poetry"
3. Do a test on "words" and you can choose any amount

**Expected behavior**  The limit should be the amount of words that you set it as.

**Screenshots**  
Screenshot 2021-08-13 at 4 41 32 PM

- OS: []  Chrome OS
- Browser []  Chrome
- Browser Version []  Version 88.0.4324.208 (Official Build) (64-bit)

**Additional context**

### PR DESCRIPTION
**Pull request by favault**
### Description

Checks if the mode is words and changes maxWords to the current word limit
Closes #1721

### CODE REVIEW

**Comment by Miodec** [comment]
Is this at all tested? It break the website completely

File: src/js/test/poetry.js [hunk 1]
```diff
@@ -1,6 +1,11 @@
 const bannedChars = ["—", "_", " "];
-const maxWords = 100;
+var maxWords = 100;
 const apiURL = "https://poetrydb.org/random";
+import Config from './config';
```
Comment (UnrealApex) [inline src/js/test/poetry.js:4]: Put the import statement at the top of the file.

File: src/js/test/poetry.js [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+
+if (Config.mode === "words") {
```
Comment (UnrealApex) [inline src/js/test/poetry.js:6]: Use a ternary operator here:
```javascript
maxWords = (Config.mode === "words" ? Config.words : maxWords);
```

File: src/js/test/poetry.js (hunk 1 above)
Comment (UnrealApex) [reply src/js/test/poetry.js:4]: Import everything from config module:
```javascript
import * as Config from './config';
```

**Review by UnrealApex — CHANGES_REQUESTED** [review]

**Comment by Miodec** [comment]
Closed by #2069

**Pull request closed without merging**